nemo/collections/llm/tools/auto_configurator/core/training_config.py

```diff
@@ -234,13 +271,12 @@ class GPT3GridSearch:
     seq_length: int
     gpu_memory_gb: int
 
-    tp = [1, 2, 4, 8]
-    pp = [1]
-    cp = [1]
-    ep = [1]
-    mbs = [1, 2, 4, 8]
-
-    gbs: int = 1024
+    tp: List[int] = field(default_factory=lambda: [1, 2, 4, 8])
+    pp: List[int] = field(default_factory=lambda: [1])
+    cp: List[int] = field(default_factory=lambda: [1])
+    ep: List[int] = field(default_factory=lambda: [1])
+    mbs: List[int] = field(default_factory=lambda: [1, 2, 4, 8])
+    gbs: List[int] = field(default_factory=lambda: [1024])
```

Comment:
Just curious - why are these lambda functions? Can they just be standard default values like how they were originally written?